tests/views_integration/Tribe/Events/Views/V2/Utils/SeparatorsTest.php

```diff
@@ -1,227 +1,73 @@
 <?php
+
 namespace Tribe\Events\Views\V2\Utils;
 
 use Tribe\Events\Test\Factories\Event;
 use Tribe__Date_Utils as Dates;
 
 class SeparatorsTest extends \Codeception\TestCase\WPTestCase {
 
-	public function month_separator_provider() {
-		$event_factory = new Event();
```

Comment:
Events created in the dataprovider are created at the start of the test suite! This refactor changes that.